src/passes/identifierManglerPass/declarationNameMangler.ts

```diff
@@ -141,5 +163,31 @@ export class DeclarationNameMangler extends ASTMapper {
     node.vFunctions.forEach((n) => this.mangleFunctionDefinition(n));
     node.vContracts.forEach((n) => this.mangleContractDefinition(n));
     this.commonVisit(node, ast);
+
+    // Checking if counter is greater than initialIdWidth digits. If so, names are
+    // modified to insert 0's to achieve id's fixed width.
+    const lastIdSize = this.lastUsedId.toString().length;
+    if (lastIdSize > this.initialIdWidth) {
+      this.nodesNameModified.forEach((node) => {
+        if (node instanceof FunctionDefinition) {
+          node.name = this.updateName(node.name, MANGLED_INTERNAL_USER_FUNCTION, lastIdSize);
+        } else if (node instanceof VariableDeclaration) {
```

Comment:
Does the `MANGLED_INTERNAL_USER_FUNCTION` should be used in both public and private functions?

Reply:
No, that condition is over the `nodesNameModified` elements. And the only nodes of type `FunctionDefinition` added to that list are internal functions.